valve_grammar.ne

```diff
@@ -39,23 +39,23 @@ arguments -> argument ("," _ argument):* {%
     return flatten(d).filter(item => item && item != ",");
   } %}
 
-argument -> field | label | int | function | regex | named_arg
+argument -> datatype | dqstring | field | function | int | named_arg | regex
 
-field -> label "." label {% function(d) {
+field -> WORD "." label {% function(d) {
```

Comment:
A sheet name in Google Sheets or Excel can contain spaces, so I was leaving room for a table name to include spaces, in which case it would have to be double-quoted.

Reply:
OK - I've reverted this change